stock_move_actual_date/models/actual_date_mixin.py

```diff
@@ -16,9 +16,52 @@ class ActualDateMixin(models.AbstractModel):
         compute="_compute_is_editable_actual_date", string="Is Editable"
     )
 
+    def _get_trigger_field_names_for_actual_date_source(self):
+        """Return a list of field names that trigger actual_date_source assignment.
+
+        Should be overridden in specific models to return relevant fields.
+        Example: ['date_done', 'move_ids'] for stock.picking.
+        """
+        return []
```

Comment:
I think it's conceptually cleaner to include `actual_date` in the returned list of this method. This is my opinion and you might think otherwise.
```suggestion
    def _get_actual_date_update_triggers(self):
        """Return a list of field names that trigger actual_date_source assignment for stock moves.

        Should be extended in specific models to return relevant fields.
        Example: Append 'date_done' and 'move_ids' for stock.picking.
        """
        return ["actual_date"]
```

Reply:
I agree.